components/ReactBKKLogo/RevealEllipse.js

```diff
@@ -44,6 +45,17 @@ class RevealEllipse extends React.Component {
       this.timer = Date.now()
       this.raf = window.requestAnimationFrame(this.revealStep)
     }, this.props.delay)
+    document.addEventListener('click', function (e) {
+      e.preventDefault()
+      if (e.target && e.target.parentNode.nodeName === 'A') {
```

Comment:
We can’t always assume that the user will click on the `<span>` inside an `<a>`. The click even can occur anywhere inside the `<a>`, even on the `<a>` tag itself. So the correct way is to use a for/while loop.